src/property-filter/interfaces.ts

```diff
@@ -215,6 +215,7 @@ export namespace PropertyFilterProps {
     tokenLimitShowMore?: string;
     tokenLimitShowFewer?: string;
     clearFiltersText: string;
+    operatorAriaLabel?: string;
```

Comment:
`operatorAriaLabel` to me is not very clear, which operator it refers to as there are many operators for different uses in PropertyFilter. I suggest more specific naming like `tokenOperatorAriaLabel`. @avinashbot wdyt?

Reply:
it made sense to me, so went ahead and fixed, @avinashbot, let me know if you see any issues.

@YueyingLu